scripts/user/add-user.ts

```diff
@@ -133,20 +117,27 @@ const sendWelcomeEmail = async ({ email }) => {
 
 // eslint-disable-next-line @typescript-eslint/no-floating-promises
 (async () => {
-  const params = {
+  const { tableName } = await getDestinationTableInfo();
+  environment.dynamoDbTableName = tableName;
+
+  const judgesChambers: JudgeChambersInfo[] = await applicationContext
+    .getUseCases()
+    .getJudgesChambersInteractor(applicationContext, mockJudgeUser);
```

Comment:
Using the `mockJudgeUser` for permissions. Sort of silly. Happy to hear better options.